feature-libs/product-configurator/rulebased/components/group-menu/configurator-group-menu.component.service.ts

```diff
@@ -62,22 +61,22 @@ export class ConfiguratorGroupMenuService {
    * @param {number} currentGroupIndex - Current group index
    * @param {number} focusedGroupIndex - Focused group index
    * @returns {number} - updated group index
-   * @protected
    */
   protected updateCurrentGroupIndex(
     currentGroupIndex: number,
-    focusedGroupIndex: number
+    focusedGroupIndex?: number
   ): number {
-    return focusedGroupIndex !== currentGroupIndex
-      ? focusedGroupIndex
+    return focusedGroupIndex
```

Comment:
Just to doouble check: Are you aware that this statement is falsy also when the index is 0? Taking it into account, is this logic correct?

Reply:
Took into account your feedback: 
- add `if` condition for readability,
- add some tests.
The **current group index** will be updated if is is not equal to the **focused group index**.
If the **focused group index** is `null`, `undefine` or `equal` to the **current group index**, nothing happens